pkgs/development/python-modules/ansible-pylibssh/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, libssh
+, cython
+, setuptools
+, toml
+, expandvars
+}:
+
+buildPythonPackage rec {
+  pname = "ansible-pylibssh";
+  version = "1.1.0";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-spaGux6dYvtUtpOdU6oN7SEn8IgBof2NpQSPvr+Zplg=";
+  };
+
+  propagatedBuildInputs = [
+    libssh
+    cython
+    setuptools
+    toml
+    expandvars
+  ];
+
+  # Tests can only be executed against an installed pylibssh package
+  doCheck = false;
```

Comment:
What error do you receive when you try to run the tests? For Python, the check phase is moved to the install check phase, and we install the built wheel first before running the tests. Since your source code is under the `src` directory, I don't expect there to be a name conflict when importing.

Reply:
I honestly don't remember what error I received anymore, but I tested it again and it doesn't appear anymore, so I have just removed these lines to set `doCheck` back to its default of `true`.